### ISSUE TITLE
Bug: CLI Issue for flexible Heredoc syntaxes

### ISSUE THREAD

**Issue by ChibuezeAgwuDennis** [codeigniter4/CodeIgniter4#7057]
### PHP Version

7.4

### CodeIgniter4 Version

4.2.11

### CodeIgniter4 Installation Method

Manual (zip or tar.gz)

### Which operating systems have you tested for this bug?

Windows

### Which server did you use?

apache

### What happened?

CodeIgniter v4.2.11 Command Line Tool - Server Time: 2023-01-06 08:41:28 UTC-06:00

[ParseError]

Invalid body indentation level (expecting an indentation level of at least 8)

at SYSTEMPATH\Commands\Database\ShowTableInfo.php:53

### Steps to Reproduce

`php spark debugbar:clea`

### Expected Output

...

I fixed this issue before but when I updated to CI 4.2.11 it appear again please fix this

**Comment by datamweb** [comment]
@ChibuezeAgwuDennis What is your exact PHP version?

```console
php -v
```

**Comment by ChibuezeAgwuDennis** [comment]
php v7.4.9

**Comment by kenjis** [comment]
Cannot reproduce:
```console
bash-3.2$ php spark debugbar:clea

CodeIgniter v4.2.11 Command Line Tool - Server Time: 2023-01-06 17:30:17 UTC-06:00

Command "debugbar:clea" not found.

Did you mean this?
    debugbar:clear

bash-3.2$ php spark debugbar:clear

CodeIgniter v4.2.11 Command Line Tool - Server Time: 2023-01-06 17:30:21 UTC-06:00

Debugbar cleared.
```

**Comment by datamweb** [comment]
I can reproduce it, with the following changes:
```php
    /**
     * the Command's usage
     *
     * @var string
     */
    protected $usage = <<<'EOL'
        db:table [<table_name>] [options]

            Examples:
            db:table --show
            db:table --metadata
            db:table my_table --metadata
            db:table my_table
            db:table my_table --limit-rows 5 --limit-field-value 10 --desc
        EOL;
```
```php
    /**
     * the Command's usage
     *
     * @var string
     */
    protected $usage = <<<'EOL'
    db:table [<table_name>] [options]

        Examples:
        db:table --show
        db:table --metadata
        db:table my_table --metadata
        db:table my_table
        db:table my_table --limit-rows 5 --limit-field-value 10 --desc
        EOL;

```

![Screenshot 2023-01-07 152542](https://user-images.githubusercontent.com/9530214/211149412-05af03d5-7bf6-47c1-a5a2-5d4b66988001.jpg)

I don't know why this is happening to you by default. Maybe because of the settings of IDE or the activation of a plugin in Editor.

**Comment by kenjis** [comment]
This is the code in v4.2.11:
https://github.com/codeigniter4/framework/blob/543dd5e5d3cadee764962712de29458b5524dd24/system/Commands/Database/ShowTableInfo.php#L46-L60

It seems no problem.

**Comment by ChibuezeAgwuDennis** [comment]
I did not change file there

**Comment by paulbalandan** [comment]
Please update to PHP v7.4.10+. This is a bug of PHP itself https://bugs.php.net/bug.php?id=79934

**Comment by paulbalandan** [comment]
Also, please update your IDE settings not to convert LF endings to CRLF.

**Comment by kenjis** [comment]
https://www.php.net/ChangeLog-7.php#PHP_7_4
> Version 7.4.10
> ...
> Fixed bug [#79934](http://bugs.php.net/79934) (CRLF-only line in heredoc causes parsing error).
